src/GraphQL.Tests/Utilities/SchemaBuilderTestBase.cs

```diff
@@ -68,7 +68,7 @@ public ExecutionResult AssertQuery(Action<ExecutionOptions> options, ExecutionRe
         public ExecutionResult CreateQueryResult(string result) => result.ToExecutionResult();
 
         protected string ReadSchema(string fileName)
-            => File.ReadAllText(Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location), "Files", fileName));
+            => File.ReadAllText(Path.Combine(Directory.GetCurrentDirectory(), "Files", fileName));
```

Comment:
Why? Current dir may be changed (`SetCurrentDirectory`), it is a mutable concept. I used `Assembly.GetExecutingAssembly().Location` specifically to ensure correct paths.

Reply:
I don't know.  `Assembly.GetExecutingAssembly().Location` gives some oddball path when running tests with .NET Framework 4.8.  Probably something peculiar about Visual Studio's test runner.  In any case, using `Directory.GetCurrentDirectory()` works properly for both .NET Framework 4.8 and .NET Core 2.1 / 3.1.  I can take the data and embed it as a resource into the assembly, or I can just copy the text into a string constant.  But `Assembly.GetExecutingAssembly().Location` doesn't work.

Reply:
It is strange. Well, let's use getcurrentdirectory, ok.